### PR TITLE
Add Pauline to CodeOwners and script to autogenerate chain info

### PR DESCRIPTION
**Pull request by albertov19**

### CODE REVIEW

**Review by eshaben — APPROVED** [review]

File: scripts/src/chains/btc.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (evgeniko) [inline scripts/src/chains/btc.json:1]: Why does it generate an empty btc json?

Comment (eshaben) [reply scripts/src/chains/btc.json:1]: These files are not auto-generated right now. This is the way it was in the previous repo: https://github.com/wormhole-foundation/docs.wormhole.com/blob/main/scripts/src/chains/btc.json

Comment (eshaben) [reply scripts/src/chains/btc.json:1]: It was also one of the skipped chains: https://github.com/wormhole-foundation/docs.wormhole.com/blob/b8e0f96dce1e535e11ed6bf6019b282d87be08bc/scripts/src/config.ts#L168

Comment (evgeniko) [reply scripts/src/chains/btc.json:1]: ok, got it. Thank you for clarifying!

**Review by evgeniko — APPROVED** [review]

**Pull request merged**